heron/common/src/java/com/twitter/heron/common/basics/TypeUtils.java

```diff
@@ -87,6 +87,17 @@ public static ByteAmount getByteAmount(Object o) {
     }
   }
 
+  public static ByteAmount getByteAmountMB(Object o) {
```

Comment:
@mycFelix actually, now I think @ashvina's suggestion is better. It would be nice to add a enum class `ByteAmountUnit` in the same style of [`SystemConfigKey`](https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/config/SystemConfigKey.java) to expose [`MB`](https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/basics/ByteAmount.java#L22) and [`GB`](https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/basics/ByteAmount.java#L23) in a typed manner.